packages/schematics/angular/migrations/update-8/drop-es6-polyfills.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { Rule, Tree } from '@angular-devkit/schematics';
+import * as ts from '../../third_party/github.com/Microsoft/TypeScript/lib/typescript';
+
+const commentsToDrop = [
+  "// import 'core-js/es6/reflect';",
+  '/** IE10 and IE11 requires the following for the Reflect API. */',
+];
+
+const header = `/**
+*/
+
+/***************************************************************************************************
+* BROWSER POLYFILLS
+*/
+`;
+
+export const dropES2015Polyfills = (): Rule => {
+  return (tree: Tree) => {
+    const path = '/polyfills.ts';
+    const source = tree.read(path);
+    if (!source) {
+      return;
+    }
+
+    // Start the update of the file.
+    const recorder = tree.beginUpdate(path);
+
+    const sourceFile = ts.createSourceFile(path, source.toString(), ts.ScriptTarget.Latest, true);
+    const imports = sourceFile.statements
+        .filter(s => s.kind === ts.SyntaxKind.ImportDeclaration) as ts.ImportDeclaration[];
+
+    if (imports.length === 0) { return; }
+
+    for (const i of imports) {
+      const module = ts.isStringLiteral(i.moduleSpecifier) && i.moduleSpecifier.text;
+      if (module && (module.startsWith('core-js/es6/') || module.startsWith('zone.js/dist/zone'))) {
```

Comment:
we should only remove the polyfills that the CLI placed in the file.  The developer may have added addition ones.